### PR TITLE
Update bug_report.md

### PR DESCRIPTION
**Pull request by jcqli**
Include app versions in bug report template. Towards https://github.com/google/ground-android/issues/2731

@gino-m  PTAL?

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -10,6 +10,11 @@ assignees: ''
 **Describe the bug**
 A clear and concise description of what the bug is.
 
+**App versions**
+Android:
+
+Web:
```
Comment (gino-m) [inline .github/ISSUE_TEMPLATE/bug_report.md:16]: Same comment as on the other related PR - there's no version in the web app atm. Even if there were, the user probably wouldn't know with which version of the web app was used to create the survey.

**Review by gino-m — CHANGES_REQUESTED** [review]
Is it possible to add the issue to the "Ground" project via this template? It would be helpful, since atm we need to first save the bug for an issue to be auto-added.

**Comment by jcqli** [comment]
> Is it possible to add the issue to the "Ground" project via this template? It would be helpful, since atm we need to first save the bug for an issue to be auto-added.

Github automation should already be doing that -- e.g. https://github.com/google/ground-android/issues/2735 
image

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (jcqli) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Removed. I'll update https://github.com/google/ground-platform/issues/2015 as well

**Comment by gino-m** [comment]
> > Is it possible to add the issue to the "Ground" project via this template? It would be helpful, since atm we need to first save the bug for an issue to be auto-added.
> 
> Github automation should already be doing that for every new issue -- e.g. #2735 image

@jcqli  It only does that after the bug is saved though; none of the project attributes are editable until then.

**Review by shobhitagarwal1612 — APPROVED** [review]

**Review by gino-m — APPROVED** [review]

**Pull request merged**